.github/workflows/website.yml

```diff
@@ -0,0 +1,27 @@
+name: Website checks
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'v[0-9]+.[0-9]+'
+      - checks-workflow-dev/*

```

Comment:
what's this branch? 👀

Reply:
I don't know, I copied it over from the other actions.